tests/e2e/utils/logger.ts

```diff
@@ -1,23 +1,23 @@
-const fs = require('fs');
-const path = require('path');
-const {LOG_FILE} = require('../config');
+import * as fs from 'fs';
+import * as path from 'path';
+import {LOG_FILE} from '../config';
 
 let isVerbose = true;
-const setLogLevelVerbose = (value) => {
+const setLogLevelVerbose = (value: boolean) => {
     isVerbose = value;
 };
 
 // On CI systems when using .progressInfo, the current line won't reset but a new line gets added
 // Which can flood the logs. You can increase this rate to mitigate this effect.
-const LOGGER_PROGRESS_REFRESH_RATE = process.env.LOGGER_PROGRESS_REFRESH_RATE || 250;
+const LOGGER_PROGRESS_REFRESH_RATE = process.env.LOGGER_PROGRESS_REFRESH_RATE ?? 250;
```

Comment:
make sure that `process.env.LOGGER_PROGRESS_REFRESH_RATE` can't be 0 becuase `??` wont work here

Reply:
good point, let's don't change the logic as it may break something